src/auth/auth.actions.js

```diff
@@ -0,0 +1,29 @@
+export const GET_USER = 'GET_USER';
+export const SET_USER = 'SET_USER';
+
+export const getUser = (data, navigate) => {
+    return {
+        type: GET_USER,
+        payload: data,
+        navigate
+    };
+};
+
+export const setUser = (user) => ({
+    type: SET_USER,
+    user
+});
+
+const initialState = {
+    user: undefined
+};
+
+export function userReducer(state = initialState, action) {
```

Comment:
reducers should be kept in separate files from your actions

Reply:
done